addons/common/stringtable.xml

```diff
@@ -351,6 +358,12 @@
         </Key>
         <Key ID="STR_ZEN_Common_Radius_Units">
```

Comment:
Additional translations apart from German were added to this entry. Can't comment on these...

Reply:
For the height it is also the normal + (m)
https://github.com/zen-mod/ZEN/blob/5f5bf12a7a2cbec0cea6575c115ed3964d8f31e0/addons/common/stringtable.xml#L379-L388